stdlib/public/Platform/glibc.freebsd.modulemap.in

```diff
@@ -16,8 +16,8 @@
 /// still be issues if imported into the same context as one in which someone
 /// included those headers directly.
 ///
-/// It's not named just FreeBSD libc so that it doesn't conflict in the event of a
-/// future official FreeBSD libc modulemap.
+/// It's not named just FreeBSD libc so that it doesn't conflict in the event of
+/// a future official FreeBSD libc modulemap.
```

Comment:
Does this comment make sense in the larger file? It uses it so many times, it bugs me.

I would prefer something like:

*NOTE* We do not name this FreedBSD libc so that we do not conflict with a potential future official FreeBSD libc module map.

Or something like that.